src/components/PhotosPicker.vue

```diff
@@ -20,51 +21,26 @@
  -
  -->
 <template>
-	<div class="photos-picker">
-		<div class="photos-picker__content">
-			<nav class="photos-picker__navigation" :class="{'photos-picker__navigation--placeholder': monthsList.length === 0}">
-				<ul>
-					<li v-for="month in monthsList"
-						:key="month"
-						class="photos-picker__navigation__month"
-						:class="{selected: targetMonth === month}">
-						<NcButton type="tertiary" :aria-label="t('photos', 'Jump to {date}', {date: dateMonthAndYear(month)})" @click="targetMonth = month">
-							{{ month | dateMonthAndYear }}
-						</NcButton>
-					</li>
-				</ul>
-			</nav>
-
-			<FilesListViewer class="photos-picker__file-list"
-				:class="{'photos-picker__file-list--placeholder': monthsList.length === 0}"
-				:file-ids-by-section="fileIdsByMonth"
-				:empty-message="t('photos', 'There are no photos or videos yet!')"
-				:sections="monthsList"
-				:loading="loadingFiles"
-				:base-height="100"
-				:section-header-height="50"
-				:scroll-to-section="targetMonth"
-				@need-content="getFiles"
-				@focusout.native="onFocusOut">
-				<template slot-scope="{file, height, isHeader, distance}">
-					<h3 v-if="isHeader"
-						:id="`photos-picker-section-header-${file.id}`"
-						:style="{ height: `${height}px`}"
-						class="section-header">
-						{{ file.id | dateMonthAndYear }}
-					</h3>
-
-					<File v-else
-						:file="files[file.id]"
-						:allow-selection="true"
-						:selected="selection[file.id] === true"
-						:distance="distance"
-						@select-toggled="onFileSelectToggle" />
-				</template>
-			</FilesListViewer>
-		</div>
-
-		<div class="photos-picker__actions">
+	<NcDialog :name="name"
+		:open="open"
+		out-transition
+		size="large"
+		@update:open="(open) => $emit('update:open', open)">
+		<!-- Navigation containing the months available -->
+		<template #navigation>
+			<ul>
+				<li v-for="month in monthsList" :key="month">
+					<NcButton :type="targetMonth === month ? 'secondary' : 'tertiary'"
+						:aria-label="t('photos', 'Jump to {date}', { date: dateMonthAndYear(month) })"
+						@click="targetMonth = month">
+						{{ dateMonthAndYear(month) }}
+					</NcButton>
+				</li>
+			</ul>
+		</template>
```

Comment:
When there are many months and viewport is 320px, it does not scroll

We should use NcSelect when in mobile view like in nc/dialogs FilePicker

Reply:
Done :heavy_check_mark: 


![localhost_8082_index php_apps_photos_albums_a](https://github.com/nextcloud/photos/assets/1855448/5bb16f52-336e-435d-b5ad-37ea7f1c26d2)